mithril-infra/assets/docker/docker-compose-signer-unverified-norelay.yaml

```diff
@@ -60,6 +60,9 @@ services:
       - CARDANO_CLI_PATH=/app/bin/cardano-cli
       - ERA_READER_ADAPTER_TYPE=${ERA_READER_ADAPTER_TYPE}
       - ERA_READER_ADAPTER_PARAMS=${ERA_READER_ADAPTER_PARAMS}
+      - ENABLE_METRICS_SERVER=${ENABLE_METRICS_SERVER}
```

Comment:
Would it be easier to identify if the name started with METRICS_SERVER like other variable for that feature ? 
It could be METRICS_SERVER_ENABLED.

Reply:
This environment var name is based on the clap arguments nomenclature where we usually prefix with `enable_` rather than suffix with `_enabled`.